src/3rdparty/walkontable/src/tableRenderer.js

```diff
@@ -375,14 +377,12 @@ class WalkontableTableRenderer {
         TD = TD.nextSibling; //http://jsperf.com/nextsibling-vs-indexed-childnodes
       }
       // If the number of headers has been reduced, we need to replace excess TH with TD
-      if (TD.nodeName == 'TH') {
+      if (TD.nodeName === 'TH') {
         TD = replaceThWithTd(TD, TR);
       }
-      if (!hasClass(TD, 'hide')) {
```

Comment:
The main idea of this improvement is not to recreate DOM element each time, but update it if needed. So firstly we update virtual element and then compare it with DOM element and update differences.